docs/build-for-developers/cli-usage.md

```diff
@@ -54,10 +55,39 @@ openfn path/to/job.js -a http=/repo/openfn/adaptors/my-http-build
 openfn path/to/job.js -ma http
 ```
 
-Set a path to the monorepo with the env var OPENFN_REPO_DIR (eg, `OPENFN_REPO_DIR=~/openfn/repo openfn job.js -ma http`).
+Set a path to the monorepo with the env var OPENFN_REPO_DIR (eg,
+`OPENFN_REPO_DIR=~/openfn/repo openfn job.js -ma http`).
```

Comment:
@josephjclark i don't think i understand this part `OPENFN_REPO_DIR=~/openfn/repo openfn job.js -ma http`).` How does this work exactly ?

Reply:
I'm not sure what's not to understand! :grimacing: 

This sets an env var (`OPENFN_REPO_DIR`) to a value before running the `openfn` command with a job, the http adaptor, and the monorepo flag.

Usually of course you'd set the env var in your `.zhrc` or whatever,

This diff is just caused by prettier - these aren't new docs or new functionality.

Reply:
@josephjclark i am only familiar with these two 
```
export OPENFN_ADAPTORS_REPO=/Users/openfn/Workspace/adaptors
export OPENFN_ALWAYS_CACHE_STEPS=true
```

I have never used `OPENFN_REPO_DIR` before, that's why i am surprised

Reply:
Oh! Right, that makes sense. Let me fix that - thanks!

Reply:
Ach, sorry, you're right! Should be good now